README.md

```diff
@@ -212,6 +219,29 @@ class ExampleComponent(TransformComponent):
         """
 ```
 
+
+###  Running your pipeline
+
+Once you have a pipeline you can easily run (and compile it) using the builtin CLI:
+
+```bash
+fondant run foo/bar.py:pipeline --local
```

Comment:
This is for the --local, should we add both?

Reply:
the kubeflow command is there but raises NotImplementedError's